ceph/ceph/templates/bin/_start_mgr.sh.tpl

```diff
@@ -31,32 +31,31 @@ fi
 
 log "SUCCESS"
 
-ceph -v 
-
-if [[ "$MGR_DASHBOARD" == 1 ]]; then
-    ceph ${CLI_OPTS} mgr module enable dashboard --force
-    ceph ${CLI_OPTS} config-key put mgr/dashboard/server_addr "$MGR_IP"
-    ceph ${CLI_OPTS} config-key put mgr/dashboard/server_port "$MGR_PORT"
-fi
-
-if [[ "$LOCAL_POOL" == 1 ]]; then
-    ceph ${CLI_OPTS} mgr module enable localpool --force
-    if [ -n "${LPOOL_FAILURE_DOMAIN}" ]; then
-        ceph ${CLI_OPTS} config-key set mgr/localpool/failure_domain ${LPOOL_FAILURE_DOMAIN}
-    fi
-    if [ -n "${LPOOL_SUBTREE}" ]; then
-        ceph ${CLI_OPTS} config-key set mgr/localpool/subtree ${LPOOL_SUBTREE}
-    fi
-    if [ -n "${LPOOL_PG_NUM}" ]; then
-        ceph ${CLI_OPTS} config-key set mgr/localpool/pg_num ${LPOOL_PG_NUM}
-    fi
-    if [ -n "${LPOOL_NUM_REP}" ]; then
-        ceph ${CLI_OPTS} config-key set mgr/localpool/num_rep ${LPOOL_NUM_REP}
-    fi
-    if [ -n "${LPOOL_MIN_SIZE}" ]; then
-        ceph ${CLI_OPTS} config-key set mgr/localpool/min_size ${LPOOL_MIN_SIZE}
-    fi
-fi
+ceph -v
+
+#vars look like module_option=value one of the
```

Comment:
This comment doesn't appear to have anything to do with the line below it and it's unclear what 'vars' you're referring to; let's make it super clear

Reply:
Also, please comment with "# " rather than just "#"

Reply:
yeah that's an old comment I forgot to remove after rebase. Will remove